### ISSUE TITLE
Multiselect on change not firing when virtualization

### ISSUE THREAD

**Issue by cferrar** [aurelia-ui-toolkits/aurelia-kendoui-bridge#794]
When using virtualization the on change event does not fire when you remove a selected item. When you select an item the change event does fire.

This works.
                <ak-multiselect k-widget.bind="clientMultiSelect"
                                k-placeholder="Client"
                                k-data-text-field="Name"
                                k-data-value-field="Code"
                                k-data-source.bind="clientDataSource"
                                k-auto-bind.bind="false"
                                k-on-change.delegate="clientMultiSelect_OnChange($event.detail)"
                                k-value.two-way="clientCode">
                </ak-multiselect>

This does not fire the change event when a selected item is removed. But it does fire when an item is selected.

                <ak-multiselect k-widget.bind="clientMultiSelect"
                                k-placeholder="Client"
                                k-data-text-field="Name"
                                k-data-value-field="Code"
                                k-data-source.bind="clientDataSource"
                                k-auto-bind.bind="false"
                                k-on-change.delegate="clientMultiSelect_OnChange($event.detail)"
                                k-value.two-way="clientCode"
                                k-virtual.bind="virtualClient">
                </ak-multiselect>

**Comment by JeroenVinke** [comment]
Hey! What version of Kendo are you on?

**Comment by cferrar** [comment]
Kendo. version returns 2018.3.1017

Thanks,
Chris

**Comment by JeroenVinke** [comment]
Hmm I can't seem to reproduce it: https://gist.run/?id=c8e986dce9650048d7b846e1a1ed62dd

Am I doing something wrong?

**Comment by cferrar** [comment]
I don't think you are missing anything but maybe I am. I copied the example exactly into my code and it mostly works as I would expect. The only thing that is odd is that when I delete the last selected item in my application I do not get the change event. If there are 2 selected items and I delete one I get the change even. I did not see this before because I was only allowing one selection on the control.

I do not see that I'm doing anything different then the gist example.

thanks,
Chris Ferrar
